File: Iniciante/URI 1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');
var x1 = parseFloat(line1[0]);
var y1 = parseFloat(line1[1]);

var line2 = lines.shift().split(' ');
var x2 = parseFloat(line2[0]);
var y2 = parseFloat(line2[1]);

var distancia = Math.sqrt(Math.pow((x2 - x1), 2.0) + Math.pow((y2 - y1), 2.0));
console.log(distancia.toFixed(4));

File: Iniciante/URI 1015.r
input = file('stdin', 'r')

line1 = strsplit(readLines(input, n=1), " ")
x1 = as.double(line1[[1]][1])
y1 = as.double(line1[[1]][2])

line2 = strsplit(readLines(input, n=1), " ")
x2 = as.double(line2[[1]][1])
y2 = as.double(line2[[1]][2])

distancia = sqrt(((x2 - x1) ** 2) + ((y2 - y1) ** 2))
write(sprintf("%.4f", distancia), '')